endpoints/api/user.py

```diff
@@ -545,6 +546,83 @@ def delete(self):
         return "", 204
 
 
+@resource("/v1/user/initialize")
+class InitializeUser(ApiResource):
```

Comment:
This api should be put behind a feature flag, and defaults to `False`

Reply:
For what purpose? It only functions on a clean db.